### DISCUSSION TITLE
[WinError 2] The system cannot find the file specified

### DISCUSSION THREAD

**Discussion by Atoli** [openai/whisper#1468] [Q&A]
Greetings.

I have an issue.
I installed whisper, however, when i try to use with via command line with an audio file, it says it cannot find said file (even if it's there).

Installed it as the main page says: `pip install git+https://github.com/openai/whisper.git `.
Attached image is the errors that pop up when run `whisper audiofilename.wav`.

What could the issue be?

OS: Windows 10.
Python ver: 3.10.

![whisper](https://github.com/openai/whisper/assets/43733214/98ab18b4-5ceb-4649-bf7e-e4bb2ef5311e)

I have read this could happen if python is not installed but i already checked pip, it says `ffmpeg-python version 0.2.0` so that could not be the problem unless this `ffmpeg` is not recognized by whisper.

**Comment by phineas-pta** [comment]
re-check with `ffmpeg -version`

**Comment by Atoli** [comment]
> re-check with `ffmpeg -version`

Greetings, thank you for answer.

Weird, i tried `ffmpeg -version` in command line but it does not recognize ffmpeg as a command.

I get `'ffmpeg' is not recognized as an internal or external command, operable program or batch file.` 

If i try `pip show ffmpeg-python` i DO get:

```
Name: ffmpeg-python
Version: 0.2.0
Summary: Python bindings for FFmpeg - with complex filtering support
Home-page: https://github.com/kkroening/ffmpeg-python
Author: Karl Kroening
Author-email: karlk@kralnet.us
License: UNKNOWN
Location: c:\python310\lib\site-packages
Requires: future
```

It says i have ffmpeg only if check with pip, checking by itself does not work.

**Comment by phineas-pta** [comment]
`ffmpeg` is not `ffmpeg-python`

whisper needs the former not the latter

**Reply by SunJR24** [reply]
hello  I have the same question.
I tried unstall ffmpeg-python and install ffmpeg  ，but it still print error

code

import whisper

model = whisper.load_model("medium")
result = model.transcribe("G:\\Work-base\\processed_3.wav")
print(result["text"])

error

Traceback (most recent call last):
  File "G:\Anaconda3\envs\T3\lib\site-packages\whisper\audio.py", line 46, in load_audio
    ffmpeg.input(file, threads=0)
AttributeError: module 'ffmpeg' has no attribute 'input'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "G:\Work-base\Colab-WorkBase\VITS-fast-fine-tuning\scripts\t5.py", line 4, in <module>
    result = model.transcribe("G:\\Work-base\\Colab-WorkBase\\VITS-fast-fine-tuning\\custom_character_voice\\VO_nahida\\processed_3.wav")
  File "G:\Anaconda3\envs\T3\lib\site-packages\whisper\transcribe.py", line 121, in transcribe
    mel = log_mel_spectrogram(audio, padding=N_SAMPLES)
  File "G:\Anaconda3\envs\T3\lib\site-packages\whisper\audio.py", line 130, in log_mel_spectrogram
    audio = load_audio(audio)
  File "G:\Anaconda3\envs\T3\lib\site-packages\whisper\audio.py", line 50, in load_audio
    except ffmpeg.Error as e:
AttributeError: module 'ffmpeg' has no attribute 'Error'

进程已结束,退出代码1

**Comment by Atoli** [comment]
> `ffmpeg` is not `ffmpeg-python`
> 
> whisper needs the former not the latter

I see.

Excuse me for the noob question, but how do i install `ffmpeg` in my system? I know how to use `ffmpeg` but when i downloaded it was just an exe file in a folder, it's not "installed" in the system, but something i just run. 

Repositories like whisper and other utilities can not use it for this reason, i guess.

**Reply by phineas-pta** [reply]
u can add the ffmpeg folder to environment variable `PATH`

something like this https://linuxhint.com/add-directory-to-path-environment-variables-windows/

**Reply by jpthiru** [reply]
The instructions are here : https://www.geeksforgeeks.org/how-to-install-ffmpeg-on-windows/ 

May be useful for someone.

**Comment by SunJR24** [comment]
I solved it

you need to use anaconda

conda install -c conda-forge ffmpeg
